assets/templates/partials/previousreleases/contents/body.tmpl

```diff
@@ -1,11 +1,15 @@
-<div class="ons-pl-grid-col">
-  <section id="summary">
-    <h2>Summary</h2>
-    <p>{{ .Description.Summary }}</p>
-  </section>
-
-  <section id="releasehistory">
-    <h2>Release history</h2>
-    {{ template "partials/previousreleases/contents/release-history" . }}
-  </section>
+<div class="ons-pl-grid-col"></div>
```

Comment:
Should this `<div>` be ended here?

Reply:
Good spot. VS Code being over exuberant in its tag completion.